main.go

```diff
@@ -144,16 +159,14 @@ func main() {
 
 	uri := cfg.GetString("amqp.uri")
 	exchange := cfg.GetString("amqp.exchange.name")
-	var err error
-	client, err = messaging.NewClient(uri, true)
+
+	publisher, err := NewDefaultJobUpdatePublisher(uri, exchange)
```

Comment:
This `NewDefaultJobUpdatePublisher` isn't defined anywhere.

Reply:
D'oh! I forgot to check in the new source file.